lib/consensus/isaac.go

```diff
@@ -153,7 +153,7 @@ func (is *ISAAC) IsVoted(b ballot.Ballot) bool {
 func (is *ISAAC) Vote(b ballot.Ballot) (isNew bool, err error) {
 	is.RLock()
 	defer is.RUnlock()
-	roundHash := b.Round().Hash()
+	roundHash := b.Round().Index()
```

Comment:
Round.`Hash()` method is not using it as a hash now.
It is the purpose to distinguish which round and height it is.
So I changed `Hash()` body because `argon2` hash function uses a lot of resources.
The name `Hash` is still better you think? Then I will change it.
@spikeekips @anarcher please give me advice :)

Reply:
Orogonally I think Hashing is cheap, but it’s not. :) This code looks good.